runner/internal/recorder/benchmark.go

```diff
@@ -44,8 +37,8 @@ func (bk Benchmark) Stats() (min, max, mean time.Duration) {
 	return min, max, sum / time.Duration(n)
 }
 
-// newReport generates and returns a Benchmark given a Run dataset.
-func newReport(records []Record, numErr int, d time.Duration) Benchmark {
+// newBenchmark generates and returns a Benchmark given a Run dataset.
+func newBenchmark(records []Record, numErr int, d time.Duration) Benchmark {
```

Comment:
Do we eventually still want to use "Benchmark"?

Reply:
(the whole file is gone)